packages/lisk-api-client/package.json

```diff
@@ -19,22 +19,46 @@
 		"node": ">=6.3 <=9.5",
 		"npm": ">=3 <=5"
 	},
-	"main": "dist/index.js",
+	"browser": "dist-browser/index.min.js",
+	"main": "dist-node/index.js",
 	"scripts": {
+		"babel": "babel src -d dist-node",
```

Comment:
We should take the opportunity to give this script the generic name `transpile`.